src/domain/hanke/edit/HankeForm.tsx

```diff
@@ -20,7 +20,7 @@ import Form2 from './HankeForm2';
 import FormNotifications from './components/FormNotifications';
 import './HankeForm.styles.scss';
 import { HANKE_SAVETYPE } from '../../types/hanke';
-import { filterEmptyContacts, isHankeEditingDisabled } from './utils';
```

Comment:
Can the method `isHankeEditingDisabled` be removed, or is it used elsewhere?

Reply:
It's not used elsewhere so removed it.